building parity-util-mem fails in windows

error[E0308]: mismatched types
  --> parity-util-mem\src\allocators.rs:80:23
   |
80 |                 HeapSize(heap, 0, ptr) as usize
   |                                   ^^^ expected enum `winapi::ctypes::c_void`, found enum `std::ffi::c_void`
   |
   = note: expected raw pointer `*const winapi::ctypes::c_void`
              found raw pointer `*const std::ffi::c_void`

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -28,3 +28,4 @@ test_script:
   - cd plain_hasher/ && cargo test --no-default-features && cd ..
   - cd parity-util-mem/ && cargo test --no-default-features && cd ..
   - cd parity-util-mem/ && cargo test --features=estimate-heapsize && cd ..
+  - cd parity-util-mem/ && cargo test && cd ..

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -26,4 +26,5 @@ test_script:
   - cd fixed-hash/ && cargo test --all-features && cd ..
   - cd uint/ && cargo test --features=std,quickcheck --release && cd ..
   - cd plain_hasher/ && cargo test --no-default-features && cd ..
+  - cd parity-util-mem/ && cargo test --no-default-features && cd ..
   - cd parity-util-mem/ && cargo test --features=estimate-heapsize && cd ..

--- a/parity-util-mem/src/allocators.rs
+++ b/parity-util-mem/src/allocators.rs
@@ -66,7 +66,7 @@ mod usable_size {
 		} else if #[cfg(target_os = "windows")] {
 
 			use winapi::um::heapapi::{GetProcessHeap, HeapSize, HeapValidate};
-            use winapi::ctypes::c_void as winapi_c_void;
+			use winapi::ctypes::c_void as winapi_c_void;
 
 			/// Get the size of a heap block.
 			/// Call windows allocator through `winapi` crate

--- a/parity-util-mem/src/allocators.rs
+++ b/parity-util-mem/src/allocators.rs
@@ -66,6 +66,7 @@ mod usable_size {
 		} else if #[cfg(target_os = "windows")] {
 
 			use winapi::um::heapapi::{GetProcessHeap, HeapSize, HeapValidate};
+            use winapi::ctypes::c_void as winapi_c_void;
 
 			/// Get the size of a heap block.
 			/// Call windows allocator through `winapi` crate
@@ -73,11 +74,11 @@ mod usable_size {
 
 				let heap = GetProcessHeap();
 
-				if HeapValidate(heap, 0, ptr) == 0 {
+				if HeapValidate(heap, 0, ptr as *const winapi_c_void) == 0 {
 					ptr = *(ptr as *const *const c_void).offset(-1);
 				}
 
-				HeapSize(heap, 0, ptr) as usize
+				HeapSize(heap, 0, ptr as *const winapi_c_void) as usize
 			}
 
 		} else if #[cfg(feature = "jemalloc-global")] {

